README.md

```diff
@@ -66,3 +66,16 @@ You should see output similar to the following:
 ```
 
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
```

Comment:
README.md is out of date above - still mentions `TODO_LIST_ID` and `DONE_LIST_ID`

Instead of documenting them here where they might go out of date, the .env.template alone is probably enough. You could add comments to it if an env var is non-obvious.

Also you added the "start" poetry script which is nice, but the README still only mentions `poetry run flask run`.